mautic/mautic
# Bugfix 12063 - 500 Error when changing tag in a Contact and clicking save&close [5.x]

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ yes]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #12063 



#### Description:

This fix if for Mautic 5.x ([4.x version is here](https://github.com/mautic/mautic/pull/12105))



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. You need a Contact that has 1 tag
3. Edit the Contact and change the tag
4. Hit Save&Close
5. You should now be able to reopen the Contact (that wasn't possible before, gave a JS error or a server error)
6. If you open the Audit log, you should see the 'old' and 'new' tag listed in the latest entry
